protzilla/data_preprocessing/plots.py

```diff
@@ -325,10 +270,9 @@ def create_anomaly_score_bar_plot(
     :param anomaly_df: pandas Dataframe that contains the anomaly score for each\
     sample, including outliers and on-outliers samples
     :param colour_outlier: hex code for colour depicting the outliers.
-    Default: PROTZILLA_DISCRETE_COLOR_OUTLIER_SEQUENCE outlier colour
+    Default: PROTZILLA_SECONDARY_COLOR
```

Comment:
Verstehe ich es richtig, dass die secondary color generell die "outlier color" ist? Wenn ja, fände ich es schön, dafür einen Kommentar in der colors.py zu haben, dann weiß man das direkt?